src/material-examples/tree-dynamic/tree-dynamic-example.ts

```diff
@@ -75,10 +74,10 @@ export class DynamicDataSource {
   /** Handle expand/collapse behaviors */
   handleTreeControl(change: SelectionChange<DynamicFlatNode>) {
     if (change.added) {
-      change.added.forEach((node) => this.toggleNode(node, true));
+      change.added.forEach(node => this.toggleNode(node, true));
     }
     if (change.removed) {
-      change.removed.reverse().forEach((node) => this.toggleNode(node, false));
+      change.removed.reverse().forEach(node => this.toggleNode(node, false));
```

Comment:
Might be a good idea to add a `slice()` before the `reverse` here since `reverse` will reverse the array in place.